src/amo/components/AddonReviewList/styles.scss

```diff
@@ -53,6 +53,10 @@
   height: 36px;
 }
 
+.AddonReviewList-header-text {
+  min-width: 50%;
```

Comment:
Rationale for this is [here](https://stackoverflow.com/questions/36150458/flex-item-overflows-container-due-to-long-word-even-after-using-word-wrap#answer-36150764)

Display flex assigns min-width as total width of element if not defined so word-wrap still overflows the container.
hence need to use this with `word-wrap: break-word`

Reply:
Huh, cool. If you could add a link to that rationale to the CSS in a comment that'd be cool.